src/components/WalletInit/PasswordStrengthIndicator.js

```diff
@@ -21,34 +20,14 @@ const messages = defineMessages({
     defaultMessage: '!!!The password needs to contain at least:',
     description: 'some desc',
   },
-  passwordMinLength: {
-    id: 'components.walletinit.passwordstrengthindicator.passwordMinLength',
-    defaultMessage: '!!!7 characters',
-    description: 'some desc',
-  },
-  passwordUpperChar: {
-    id: 'components.walletinit.passwordstrengthindicator.passwordUpperChar',
-    defaultMessage: '!!!1 uppercase letter',
-    description: 'some desc',
-  },
-  passwordLowerChar: {
-    id: 'components.walletinit.passwordstrengthindicator.passwordLowerChar',
-    defaultMessage: '!!!1 lowercase letter',
-    description: 'some desc',
-  },
-  passwordNumber: {
-    id: 'components.walletinit.passwordstrengthindicator.passwordNumber',
-    defaultMessage: '!!!1 number',
-    description: 'some desc',
-  },
   continueButton: {
     id: 'components.walletinit.passwordstrengthindicator.continueButton',
     defaultMessage: '!!!Continue',
     description: 'some desc',
   },
   passwordBigLength: {
     id: 'components.walletinit.passwordstrengthindicator.passwordBigLength',
-    defaultMessage: '!!!12 characters',
+    defaultMessage: '!!!10 characters',
     description: 'some desc',
   },
   or: {
```

Comment:
This one also seems unnecessary now (sorry 😬)

Reply:
How is it unnecessary? It's used to show the user the current password requirement is 10 characters.

Reply:
I mean the "or" string, I believe it's not used anymore.

Reply:
A good point. Removed some more dead code